beast/physicsmodel/stars/stellib.py

```diff
@@ -31,9 +31,13 @@
     'elodie_3.1': __ROOT__ + '/libs/Elodie_v3.1.grid.fits',
     'kurucz': __ROOT__ + '/libs/kurucz2004.grid.fits',
     'tlusty': __ROOT__ + '/libs/tlusty.lowres.grid.fits'
```

Comment:
Missing a ',' that causes an error.  See the testing results at:
https://travis-ci.org/BEAST-Fitting/beast/builds/289351508?utm_source=github_status&utm_medium=notification